### ISSUE TITLE
Hide the “Home Strip” when running the game on iOS

### ISSUE THREAD

**Issue by C29H25N3O5** [26F-Studio/Techmino#276]
Many full-screen games on iOS can automatically hide the Home Strip once they are running. To return to the Home Screen, user have to swipe up from the bottom of the screen twice to avoid mistouching. 

Currently, this feature is not present on Techmino. Also notice that the home strip also overlapped with the build number at the bottom of the page. 

![475C3A7F-BBD0-4526-9162-BB3BB7326D0D](https://user-images.githubusercontent.com/87485746/131663395-b814cb7c-acca-4838-9b85-1528da1dbd5d.png)

**Comment by Trebor-Huang** [comment]
That's probably the exclusive fullscreen functionality. It is undesirable on PC's, which is persumably why MrZ didn't use it.So we have to do a system-dependent `conf.lua` adjustment now.发件人: C29H25N3O5 ***@***.***>发送时间: 2021年9月1日星期三 下午7:30收件人: 26F-Studio/Techmino抄送: Subscribed主题: [26F-Studio/Techmino] Hide the “Home Strip” when running the game on iOS (#276)
Many full-screen games on iOS can automatically hide the Home Strip once they are running. To return to the Home Screen, user have to swipe up from the bottom of the screen twice to avoid mistouching.
Currently, this feature is not present on Techmino. Also notice that the home strip also overlapped with the build number at the bottom of the page.


—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/26F-Studio/Techmino/issues/276",
"url": "https://github.com/26F-Studio/Techmino/issues/276",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by MrZ626** [comment]
have you tried the "fullscreen" setting?

**Comment by C29H25N3O5** [comment]
> have you tried the "fullscreen" setting?

Yeah, but it seems that the “Fullscreen” setting is not working besides changing the background a little bit :(

![CF729B83-8111-40FD-AB78-B2AD5313E243](https://user-images.githubusercontent.com/87485746/131703994-5962d3d0-ae68-41b4-b438-ba0da6a877ca.jpeg)

**Comment by MrZ626** [comment]
I tried love.window.setFullscreen(false/true,''exclusive/desktop") on iPad pro, none of them can hide the "home strip"

**Comment by C29H25N3O5** [comment]
> I tried love.window.setFullscreen(false/true,''exclusive/desktop") on iPad pro, none of them can hide the "home strip"

Maybe there is a issue with LÖVE…

**Comment by Trebor-Huang** [comment]
https://developer.apple.com/forums/thread/92232

**Comment by C29H25N3O5** [comment]
How about moving the version number aside so it won't overlap with the bar
